### DISCUSSION TITLE
Can perform the inference by slicing only the back of the image?

### DISCUSSION THREAD

**Discussion by ingbeeedd** [obss/sahi#654] [Q&A] [answered]
Hello!
If look at the small vehicles in the example image, there are many small images distributed behind the image
I'd like to apply `sahi` only to the rear horizontal part, is it possible? Is there anything wrong with the post-processing?

What do you think about this because I think it can save time on certain domain images?

Thank you!

**Comment by fcakyon** [comment, marked as answer]
Hello @ingbeeedd, that's a great question!

Yes, `sahi` supports that! 
1. You can crop the region of interest from the original image,
2. then perform sliced inference on that region 
3. then project the predicted boxes back to original 2d coordinate space 👍
